cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -56,4 +64,66 @@ describe('Dashboard', () => {
       });
     });
   });
+
+  describe('Textbox', () => {
```

Comment:
```suggestion
  context('Editing a dashboard', () => {
```

This way the overview: We are inside the Dashboard tests -> We are editing a dashboard
Editing a dashboard requires a dashboard, so we create one in the `before` hook.

I haven't tested this syntax yet (actually just discovered it when searching stuff in docs), but we need to use sth like this:

```javascript
context('editing a dashboard', () => {
  before(() => {
    createDashboardWithApiOrWhatever().as('dashboard');
  });
    
  beforeEach(() => {
    cy.visit(`/dashboards/${this.dashboard}`);
    // commands to open the edit state
  });
  
  // ...
});
```

Then, inside the test blocks we'll be able to use `this.dashboard` when needed

Reply:
Tested, it works with `this`. A few notes from my tests:
- it's necessary to use the `function(){}` syntax for it to work
- `cy.get('@dashboard')` also works, but it will have an issue with the `before` hook. All aliases are reset before each test block, so you'd only have access to it in the first test block. Anyway, with `this.dashboard` I didn't have this issue :man_shrugging:

Reply:
Omg @gabrieldutra the code is much more awesome now! 10x 🤘
It even shaved off 8 secs 😁

Notice I also refactored the create/archive tests - separated to individual tests, where the archive uses the API ;)

Reply:
I wish I could also utilize a `AddTextboxByAPI` but it requires some initial data (widget position calculation) which would complicate things. I'm aiming to change that when refactoring the widgets.